django-stubs/http/response.pyi

```diff
@@ -62,7 +62,10 @@ class HttpResponseBase(Iterable[Any]):
     def __iter__(self) -> Iterator[Any]: ...
 
 class HttpResponse(HttpResponseBase):
-    content: Any
+    @property
+    def content(self) -> bytes: ...
+    @content.setter
+    def content(self, value: Any): ...
```

Comment:
Shouldn't `setter` be annotated as `-> None`?

Reply:
Yeah, they should be there. I've added another commit for that.